packages/forma-36-react-components/src/components/Pill/Pill.css

```diff
@@ -9,6 +9,7 @@
   border-radius: 0.125rem;
   border: none;
   padding: 0;
+  justify-content: space-between;
```

Comment:
@suevalov Wondering why this is needed?

Reply:
Removing it doesn't appear to change anything in Chrome

Reply:
To make sure that the close button is always at the right corner. I might be wrong :)

Reply:
@m10l removed